data/example_realization_config_w_bmi_c__macos.json

```diff
@@ -46,27 +46,61 @@
         "cat-27": {
             "formulations": [
                 {
-                    "name": "bmi_c",
+		    "name": "bmi_multi",
```

Comment:
Same as above.... one way or another though, this file can actually be removed since #430.

Reply:
Similar to the above comment, we should have the same TODO clean up this file and make the other one OS-agnostic.